Hellanmaa Winter Units

TWatson1162 opened the pull request.

TWatson1162 committed:
Hellenma Winter Units

BrettMayson commented on addons/polkbaskera/CfgVehicles_Backpacks.hpp:
```diff
@@ -0,0 +1,6 @@
+	class CLASS(B_AssaultPack_rgr_AT): B_AssaultPack_rgr {
+		class TransportWeapons {};
+		class TransportItems {};
+		class TransportMagazines {RPG7_F,2};
```
```suggestion
		class TransportMagazines {
			class _xx_RPG7_F {
				magazine = "RPG7_F";
				count = 2;
			};
		};
```

BrettMayson commented on addons/polkbaskera/CfgVehicles_Units.hpp:
```diff
@@ -0,0 +1,93 @@
+	Items[] = {R4("ACE_elasticBandage"), R4("ACE_packingBandage"), R2("ACE_quikclot")};
+	RespawnItems[] = {R4("ACE_elasticBandage"), R4("ACE_packingBandage"), R2("ACE_quikclot")};
+	uniformClass = "CUP_I_B_PMC_Unit_29";
+	Backpack = "B_AssaultPack_rgr_AT";
```
```suggestion
	Backpack = QCLASS(B_AssaultPack_rgr_AT);
```

BrettMayson replied:
Use the backpack you've defined with the AT rounds
```suggestion
	backpack = QCLASS(B_AssaultPack_rgr_AT);
```

BrettMayson commented on addons/polkbaskera/CfgVehicles_Units.hpp:
```diff
@@ -0,0 +1,93 @@
+	role = "MissileSpecialist";
+	weapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	respawnWeapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	magazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
+	respawnMagazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
```
Only need to add one, for the weapon
```suggestion
	magazines[] = {R5("30Rnd_762x39_Mag_Green_F"), "RPG7_F", R2("SmokeShell")};
	respawnMagazines[] = {R5("30Rnd_762x39_Mag_Green_F"), "RPG7_F", R2("SmokeShell")};
```

BrettMayson commented on addons/polkbaskera/CfgVehicles_Backpacks.hpp:
```diff
@@ -0,0 +1,6 @@
+	class B_AssaultPack_rgr;
+	class CLASS(B_AssaultPack_rgr_AT): B_AssaultPack_rgr {
```
```suggestion
	class CLASS(B_AssaultPack_rgr_AT): B_AssaultPack_rgr {
		dlc = QUOTE(PREFIX); 
		scope = 1; 
		scopeCurator = 1;
```

TWatson1162 committed:
3/4 done w/ PB Units + Kardishev add

bing bong

TWatson1162 committed:
backpack hell

TWatson1162 committed:
bruh

TWatson1162 committed:
bruh_2

TWatson1162 commented:
ngl not sure what I did, but I did correctly apply all the changes requested -- however I tried to copy it for other roles (ex: CLS, Demo, Engi)

TWatson1162 commented:
yah that didn't work

TWatson1162 committed:
last push where I mess with backpacks for the time being -- I swaer

TWatson1162 committed:
P.B. Sniper

TWatson1162 commented:
k last push of the night, no more stupid piecemeal updates

TWatson1162 committed:
fixed build not working

TWatson1162 committed:
k wish it had given me that error earlier, but fixed

TWatson1162 committed:
OK FINALLY DONE

TWatson1162 committed:
k this has gotta be it -- AND IT IS -- double checked

TWatson1162 committed:
All the stuff I changed and didn't push before I left for vacation + Fixes to some of that stuff

TWatson1162 committed:
I'm gonna kermit

not even sure what i've done but it all p much works except for the MilGP stuff

TWatson1162 committed:
Adding theseus stuff

- Adding Theseus Main + Units to load order
- Putting Theseus Units as dependency for KF
- Changing KF inherit from CSAT to TACU generic OpFor

TWatson1162 committed:
Finish PB + KF + UNA, Start IDAP (Winter)

TWatson1162 committed:
IDAP

TWatson1162 committed:
Re-add helmet to PB Rifleman

TWatson1162 committed:
ACTUALLY Re-add helmet to PB Rifleman

BrettMayson committed:
format, add ranks

TWatson1162 committed:
config unit/vic/weapon/backpack additions

BrettMayson committed:
comma

BrettMayson committed:
una winter group

TWatson1162 committed:
Adding Backpacks (Can't test so mayhaps I goofed something)

TWatson1162 committed:
Final touches on dudes

just need the toolkit for the PB engineer, then done

TWatson1162 committed:
Final Addition (hopefully)

adding toolkit to engi on PB

TWatson1162 committed:
FINAL UPDATE
